view/adminhtml/templates/customer-certificates.phtml

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+/** @var \ClassyLlama\AvaTax\Block\Adminhtml\CustomerCertificates $block */
+?>
+<?php if ($block->shouldShowWarning()): ?>
+    <p class="message">
+        Please note that you can only view certificates for a customer from the website/store that customer
```

Comment:
@LordZardeck - All text needs to be run through the translate function.  Also, "credentials" is spelled incorrectly.

Reply:
@LordZardeck - Given that this is a particularly long paragraph of text, another option would be to put it in a "hidden" config path in etc/config.xml and create a block method that retrieves this.  The goal is simply to avoid having text that's hard-coded in a template and can't be changed without touching the template.

Reply:
@LordZardeck - Discussed with the senior engineers, and David pointed out that using the config option approach would be cleaner from a readability standpoint, it would be less usable when you think about SIs using language packs to translate sites for clients.  So we should stick with passing the whole thing to the translate function.